src/style.css

```diff
@@ -1,3 +1,35 @@
-body {
-  margin: 0;
+.body {
+  font-family: Roboto, sans-serif;
+}
+
+nav {
+  font-size: 18px;
+  margin-top: 10px;
+  margin-right: 10px;
+  text-align: center;
+
+}
```

Comment:
Don't use tag names for styling (except `html` and `body`). Style all elements using `.class` and if needed with `:pseudo-class`, `pseudo-element` and `[attribute]`. You should add a class to the `nav` element in your HTML and use that class for styling.

Reply:
fixed